src/components/stack/Stack.tsx

```diff
@@ -1,27 +1,36 @@
-import React, { ReactNode } from "react";
+import React, { PropsWithChildren } from "react";
 import { View, ViewStyle } from "react-native";
 import { IOSpacer } from "../../core";
 
 type AllowedStyleProps = Pick<
   ViewStyle,
-  "alignItems" | "flexShrink" | "flexGrow" | "flex" | "flexWrap"
+  "alignItems" | "flexShrink" | "flexGrow" | "flex" | "flexWrap" | "width"
 >;
 
-type Stack = {
+type Stack = PropsWithChildren<{
   space?: IOSpacer;
-  children: ReactNode;
   style?: AllowedStyleProps;
+}>;
+
+type BaseStack = Stack & {
+  orientation: "vertical" | "horizontal";
```

Comment:
That type export seems quite useless even in the `Spacer` component, so I'd not link them even if they're basically the same. But it's a good catch because I'd take advantage of this refactor to remove the export from `SpacerOrientation`